src/mds/SessionMap.h

```diff
@@ -393,6 +421,12 @@ class SessionMapStore {
   version_t version;
   ceph::unordered_map<entity_name_t, Session*> session_map;
   PerfCounters *logger;
+
+  // total request load avg
+  DecayCounter total_load_avg;
+  double get_decay_rate() const {
```

Comment:
Why not load it once when constructing SessionMap and then update it in `handle_conf_change`? See also #23139.

Reply:
yeh -- sure.